src/Queues/SqsQueue.php

```diff
@@ -18,7 +17,8 @@
  * @author Petra Barus <petra.barus@gmail.com>
  * @since 2015.02.24
  */
-class SqsQueue extends Queue {
+class SqsQueue extends Queue
```

Comment:
change the namespace to `namespace UrbanIndo\Yii2\Queue\Queues;`
Queue class is located on this path `Urbanindo\Yii2\Queue\Queue;`
so, please add `use Urbanindo\Yii2\Queue\Queue;`